projects/packages/my-jetpack/_inc/components/product-detail-card/index.jsx

```diff
@@ -210,6 +210,19 @@ const ProductDetailCard = ( { slug, onClick, trackButtonClick, className } ) =>
 						<Text>{ __( 'Active on your site', 'jetpack-my-jetpack' ) }</Text>
 					</div>
 				) }
+
+				{ 'search' === slug && (
+					<Text className={ styles[ 'search-price-disclaimer' ] } variant="body-extra-small">
+						{ __(
+							'Pricing will automatically adjust based on the number of records in your search index.',
+							'jetpack-my-jetpack'
+						) }{ ' ' }
+						{ __(
+							"If you grow into a new pricing tier, we'll let you know before your next billing cycle.",
+							'jetpack-my-jetpack'
+						) }
```

Comment:
Why do we compose the string in this way? Is it not possible to use only one? Something like the following...

```suggestion
						{ __(
							"Pricing will automatically adjust based on the number of records in your search index. If you grow into a new pricing tier, we'll let you know before your next billing cycle.",
							'jetpack-my-jetpack'
						) }
```

Reply:
I was hoping to be able to reuse the existing translations for these two separate sentences from the Jetpack Search package, but it looks like it won't work as I was hoping after looking at translate.wordpress.org. I'll go ahead and merge these two sentences into one translation call.